.changeset/poor-rules-compare.md

```diff
@@ -0,0 +1,7 @@
+---
+"@hono/sentry": patch
```

Comment:
`patch` should be `minor`.

Reply:
https://github.com/honojs/middleware/pull/620/commits/6b2ae0fbc77425efc327fb033a027f1610f8cec2